### ISSUE TITLE
Links to Glossary Broken 

### ISSUE THREAD

**Issue by mellowarrior** [Amartya/TunePad#108]
browser: Chrome Version 73.0.3683.86 
Mac OS: 10.10.5
Metronome TuneTurial: https://tunepad.live/learn/tunetorials/489
Although my internet connection is solid, all glossary links are receiving an error. 
Thanks!
<img width="852" alt="Screen Shot 2019-03-27 at 3 18 28 PM" src="https://user-images.githubusercontent.com/48967312/55109425-d716d580-50a3-11e9-8298-befaff11d294.png">

**Comment by Amartya** [comment]
Thank you for the detailed bug report. The screenshot was especially helpful (if you notice, the address bar says localhost:8888 instead of tunepad.live). 
This should now be fixed, take a look when you get a chance, and if all seems to be well, then close this issue :).

**Comment by mellowarrior** [comment]
It's fine now!  Was it something on my end?

**Comment by Amartya** [comment]
Nope! We had the wrong link in our database. I just had to go in and edit it.

In detail: that localhost link that you saw meant that we had linked to the glossary item on our local machine (where we develop the application before deploying it to an external seever) instead of the server that tunepad.live runs on. Yup, pretty boneheaded, but I m very happy you found it 😃😃😃😃.

**Comment by mellowarrior** [comment]
Considering all of the amazing coding you’ve been doing on this project, I dare say you are not allowed to use the term ‘boneheaded’ as a descriptor!
